system-tests/test_filewriter_stop_time.py

```diff
@@ -33,20 +35,20 @@ def test_filewriter_clears_stop_time_between_jobs(docker_compose_stop_command):
         message = str(message.value(), encoding="utf-8")
         if (
             '"code":"START"' in message
-            and '"job_id":"a8e31c99-8df9-4123-8060-2e009d84a0df"' in message
+            and f'"job_id":"{job_id}"' in message
         ):
             started = True
         if (
             '"code":"CLOSE"' in message
-            and '"job_id":"a8e31c99-8df9-4123-8060-2e009d84a0df"' in message
+            and f'"job_id":"{job_id}"' in message
```

Comment:
should possibly state in the systests readme that python 3.6 or above is required if format strings are to be used

Reply:
personally i think we should make this a requirement anyway, and developing or running anything under python 3.6 should be appended to the 7 deadly sins